zhaquirks/xiaomi/__init__.py

```diff
@@ -302,14 +308,14 @@ def _update_attribute(self, attrid, value):
                 "update_battery_percentage",
                 attributes[BATTERY_PERCENTAGE_REMAINING_ATTRIBUTE],
             )
-        if SMOKE in attributes:
+        if "smoke" in attributes:
```

Comment:
I'm curious, why not use the `SMOKE` constant here?

Reply:
It's temporary. Z2M has a bunch of duplicated stuff (you'll see this further below in the code) that I now duplicated and I didn't want to make a bunch of constants yet not knowing which ones we keep in the end.